spec/unit/rom/support/inflector_spec.rb

```diff
@@ -34,22 +34,68 @@
   subject(:api) { ROM::Inflector }
 
   context 'with detected inflector' do
-    it_behaves_like 'an inflector'
+    before do
+      if api.instance_variables.include?(:@inflector)
+        api.remove_instance_variable(:@inflector)
+      end
+    end
+
+    it 'prefers ActiveSupport::Inflector' do
+      expect(api.inflector == ::ActiveSupport::Inflector).to be true
```

Comment:
I believe `expect(api.inflector).to be(ActiveSupport::Inflector)` is more idiomatic rspec